### PR TITLE
chore: Remove react-infobutton in favor of keeping only react-infolabel

### PR DESCRIPTION
**Pull request by sopranopillow**
Since `react-infobutton` has been deprecated and is no longer in use, it is confusing to know which infobutton files are the correct ones for users that don't have insight on its history. Now that we have `react-infolabel` we should only keep the package that's currently in use. 

> While we are removing this, `react-components` will still re-export the package since removing it would break semver even though it's under `/unstable`.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">76</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">74</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-react-components/Avatar-mount_base.html">631</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-react-components/Avatar-mount.html">640</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-react-components/Button-mount_base.html">310</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-react-components/Button-mount.html">302</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-react-components/Field-mount_base.html">1155</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-react-components/Field-mount.html">1115</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-react-components/FluentProvider-mount_base.html">725</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-react-components/FluentProvider-mount.html">710</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-react-components/FluentProviderWithTheme-mount_base.html">77</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-react-components/FluentProviderWithTheme-mount.html">76</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">35</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">40</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">76</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">74</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-react-components/MakeStyles-mount_base.html">856</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-react-components/MakeStyles-mount.html">871</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-react-components/Persona-mount_base.html">1758</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-react-components/Persona-mount.html">1761</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-react-components/SpinButton-mount_base.html">1386</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-react-components/SpinButton-mount.html">1383</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## Perf Analysis (`@fluentui/react-northstar`)
:warning: 2 potential perf regressions detected

### Potential regressions comparing to master

Scenario | Current PR Ticks | Baseline Ticks | Ratio | Regression Analysis
:--- | ---:| ---:| ---: | ---: 
AttachmentMinimalPerf.default | [74](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/AttachmentMinimalPerf.default.html) | [76](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/AttachmentMinimalPerf.default_base.html) | 0.97:1 | [analysis](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/AttachmentMinimalPerf.default.regression.txt)
AccordionMinimalPerf.default | [76](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/AccordionMinimalPerf.default.html) | [84](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/AccordionMinimalPerf.default_base.html) | 0.9:1 | [analysis](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/AccordionMinimalPerf.default.regression.txt)
<details><summary>Perf tests with no regressions</summary>

Scenario | Current PR Ticks | Baseline Ticks | Ratio
:--- | ---:| ---:| ---:
AvatarMinimalPerf.default | [116](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/AvatarMinimalPerf.default.html) | [102](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/AvatarMinimalPerf.default_base.html) | 1.14:1
GridMinimalPerf.default | [202](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/GridMinimalPerf.default.html) | [181](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/GridMinimalPerf.default_base.html) | 1.12:1
TreeWith60ListItems.default | [89](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/TreeWith60ListItems.default.html) | [81](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/TreeWith60ListItems.default_base.html) | 1.1:1
TextMinimalPerf.default | [203](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/TextMinimalPerf.default.html) | [188](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/TextMinimalPerf.default_base.html) | 1.08:1
FlexMinimalPerf.default | [157](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/FlexMinimalPerf.default.html) | [147](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/FlexMinimalPerf.default_base.html) | 1.07:1
ListNestedPerf.default | [325](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ListNestedPerf.default.html) | [304](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ListNestedPerf.default_base.html) | 1.07:1
AttachmentSlotsPerf.default | [657](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/AttachmentSlotsPerf.default.html) | [617](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/AttachmentSlotsPerf.default_base.html) | 1.06:1
ButtonMinimalPerf.default | [88](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ButtonMinimalPerf.default.html) | [83](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ButtonMinimalPerf.default_base.html) | 1.06:1
InputMinimalPerf.default | [550](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/InputMinimalPerf.default.html) | [520](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/InputMinimalPerf.default_base.html) | 1.06:1
RefMinimalPerf.default | [110](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/RefMinimalPerf.default.html) | [104](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/RefMinimalPerf.default_base.html) | 1.06:1
TooltipMinimalPerf.default | [1293](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/TooltipMinimalPerf.default.html) | [1224](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/TooltipMinimalPerf.default_base.html) | 1.06:1
ButtonSlotsPerf.default | [324](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ButtonSlotsPerf.default.html) | [308](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ButtonSlotsPerf.default_base.html) | 1.05:1
FormMinimalPerf.default | [219](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/FormMinimalPerf.default.html) | [208](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/FormMinimalPerf.default_base.html) | 1.05:1
LayoutMinimalPerf.default | [200](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/LayoutMinimalPerf.default.html) | [190](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/LayoutMinimalPerf.default_base.html) | 1.05:1
StatusMinimalPerf.default | [403](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/StatusMinimalPerf.default.html) | [382](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/StatusMinimalPerf.default_base.html) | 1.05:1
AnimationMinimalPerf.default | [302](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/AnimationMinimalPerf.default.html) | [289](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/AnimationMinimalPerf.default_base.html) | 1.04:1
DropdownMinimalPerf.default | [1459](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/DropdownMinimalPerf.default.html) | [1405](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/DropdownMinimalPerf.default_base.html) | 1.04:1
PortalMinimalPerf.default | [87](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/PortalMinimalPerf.default.html) | [84](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/PortalMinimalPerf.default_base.html) | 1.04:1
SkeletonMinimalPerf.default | [197](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/SkeletonMinimalPerf.default.html) | [189](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/SkeletonMinimalPerf.default_base.html) | 1.04:1
SliderMinimalPerf.default | [743](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/SliderMinimalPerf.default.html) | [714](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/SliderMinimalPerf.default_base.html) | 1.04:1
CheckboxMinimalPerf.default | [1153](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/CheckboxMinimalPerf.default.html) | [1119](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/CheckboxMinimalPerf.default_base.html) | 1.03:1
SegmentMinimalPerf.default | [191](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/SegmentMinimalPerf.default.html) | [185](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/SegmentMinimalPerf.default_base.html) | 1.03:1
SplitButtonMinimalPerf.default | [2265](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/SplitButtonMinimalPerf.default.html) | [2191](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/SplitButtonMinimalPerf.default_base.html) | 1.03:1
TableMinimalPerf.default | [235](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/TableMinimalPerf.default.html) | [229](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/TableMinimalPerf.default_base.html) | 1.03:1
VideoMinimalPerf.default | [452](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/VideoMinimalPerf.default.html) | [438](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/VideoMinimalPerf.default_base.html) | 1.03:1
BoxMinimalPerf.default | [193](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/BoxMinimalPerf.default.html) | [189](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/BoxMinimalPerf.default_base.html) | 1.02:1
ListMinimalPerf.default | [304](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ListMinimalPerf.default.html) | [298](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ListMinimalPerf.default_base.html) | 1.02:1
MenuMinimalPerf.default | [510](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/MenuMinimalPerf.default.html) | [499](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/MenuMinimalPerf.default_base.html) | 1.02:1
ProviderMinimalPerf.default | [201](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ProviderMinimalPerf.default.html) | [197](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ProviderMinimalPerf.default_base.html) | 1.02:1
ReactionMinimalPerf.default | [218](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ReactionMinimalPerf.default.html) | [214](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ReactionMinimalPerf.default_base.html) | 1.02:1
IconMinimalPerf.default | [388](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/IconMinimalPerf.default.html) | [380](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/IconMinimalPerf.default_base.html) | 1.02:1
TreeMinimalPerf.default | [483](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/TreeMinimalPerf.default.html) | [475](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/TreeMinimalPerf.default_base.html) | 1.02:1
CardMinimalPerf.default | [310](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/CardMinimalPerf.default.html) | [308](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/CardMinimalPerf.default_base.html) | 1.01:1
DatepickerMinimalPerf.default | [3530](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/DatepickerMinimalPerf.default.html) | [3502](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/DatepickerMinimalPerf.default_base.html) | 1.01:1
DialogMinimalPerf.default | [444](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/DialogMinimalPerf.default.html) | [441](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/DialogMinimalPerf.default_base.html) | 1.01:1
DropdownManyItemsPerf.default | [380](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/DropdownManyItemsPerf.default.html) | [377](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/DropdownManyItemsPerf.default_base.html) | 1.01:1
ImageMinimalPerf.default | [224](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ImageMinimalPerf.default.html) | [221](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ImageMinimalPerf.default_base.html) | 1.01:1
ListWith60ListItems.default | [373](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ListWith60ListItems.default.html) | [371](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ListWith60ListItems.default_base.html) | 1.01:1
TextAreaMinimalPerf.default | [284](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/TextAreaMinimalPerf.default.html) | [280](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/TextAreaMinimalPerf.default_base.html) | 1.01:1
CustomToolbarPrototype.default | [1479](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/CustomToolbarPrototype.default.html) | [1463](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/CustomToolbarPrototype.default_base.html) | 1.01:1
ChatDuplicateMessagesPerf.default | [148](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ChatDuplicateMessagesPerf.default.html) | [148](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ChatDuplicateMessagesPerf.default_base.html) | 1:1
DividerMinimalPerf.default | [204](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/DividerMinimalPerf.default.html) | [205](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/DividerMinimalPerf.default_base.html) | 1:1
HeaderSlotsPerf.default | [455](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/HeaderSlotsPerf.default.html) | [454](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/HeaderSlotsPerf.default_base.html) | 1:1
ItemLayoutMinimalPerf.default | [700](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ItemLayoutMinimalPerf.default.html) | [700](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ItemLayoutMinimalPerf.default_base.html) | 1:1
LabelMinimalPerf.default | [224](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/LabelMinimalPerf.default.html) | [224](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/LabelMinimalPerf.default_base.html) | 1:1
MenuButtonMinimalPerf.default | [946](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/MenuButtonMinimalPerf.default.html) | [943](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/MenuButtonMinimalPerf.default_base.html) | 1:1
PopupMinimalPerf.default | [342](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/PopupMinimalPerf.default.html) | [342](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/PopupMinimalPerf.default_base.html) | 1:1
ProviderMergeThemesPerf.default | [653](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ProviderMergeThemesPerf.default.html) | [651](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ProviderMergeThemesPerf.default_base.html) | 1:1
RadioGroupMinimalPerf.default | [265](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/RadioGroupMinimalPerf.default.html) | [264](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/RadioGroupMinimalPerf.default_base.html) | 1:1
TableManyItemsPerf.default | [1099](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/TableManyItemsPerf.default.html) | [1098](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/TableManyItemsPerf.default_base.html) | 1:1
ChatMinimalPerf.default | [429](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ChatMinimalPerf.default.html) | [434](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ChatMinimalPerf.default_base.html) | 0.99:1
EmbedMinimalPerf.default | [1850](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/EmbedMinimalPerf.default.html) | [1865](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/EmbedMinimalPerf.default_base.html) | 0.99:1
ListCommonPerf.default | [383](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ListCommonPerf.default.html) | [385](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ListCommonPerf.default_base.html) | 0.99:1
CarouselMinimalPerf.default | [256](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/CarouselMinimalPerf.default.html) | [260](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/CarouselMinimalPerf.default_base.html) | 0.98:1
HeaderMinimalPerf.default | [201](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/HeaderMinimalPerf.default.html) | [206](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/HeaderMinimalPerf.default_base.html) | 0.98:1
RosterPerf.default | [1539](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/RosterPerf.default.html) | [1575](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/RosterPerf.default_base.html) | 0.98:1
AlertMinimalPerf.default | [152](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/AlertMinimalPerf.default.html) | [157](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/AlertMinimalPerf.default_base.html) | 0.97:1
ButtonOverridesMissPerf.default | [646](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ButtonOverridesMissPerf.default.html) | [667](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ButtonOverridesMissPerf.default_base.html) | 0.97:1
ChatWithPopoverPerf.default | [194](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ChatWithPopoverPerf.default.html) | [200](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ChatWithPopoverPerf.default_base.html) | 0.97:1
LoaderMinimalPerf.default | [184](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/LoaderMinimalPerf.default.html) | [190](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/LoaderMinimalPerf.default_base.html) | 0.97:1
ToolbarMinimalPerf.default | [542](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ToolbarMinimalPerf.default.html) | [568](https://fluentuipr.z22.web.core.windows.net/pull/31007/perf-test-northstar/ToolbarMinimalPerf.default_base.html) | 0.95:1

</details><div id="perfComment9424"/>

**Comment by fabricteam** [comment]
## 🕵 FluentUIV0 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

File: .github/CODEOWNERS
Comment (fabricteam) [inline .github/CODEOWNERS:222]: ## [🕵 FluentUIV0 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31007&commitId=b2360076b73216e4560f74614ea7eb145c15529b&env=prod&clientType=FLUENTUI)


  <details><summary><b>Menu</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu.Menu Example Divider Horizontal.chromium.png|76| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>

**Comment by Hotell** [comment]
wondering what happened to all these snapshots with tabster ? this should not be part of this PR

File: yarn.lock
Comment (Hotell) [inline yarn.lock:1975]: wondering if we should do the same for "@fluentui/react-alert": "9.0.0-beta.115", cc @ling1726

File: yarn.lock
Comment (Hotell) [inline yarn.lock:1987]: ATM this dedupes nicely , but this will change with any major bump of those packages. 

probably more brainstorming should be put into this how to approach this
- my immediate idea would be to completely template the `unstable/index.ts` file and add these unstable dependencies only during pre-publish step during release

**Comment by fabricteam** [comment]
## 🕵 FluentUIV0 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>

**Comment by sopranopillow** [comment]
Looks like this breaks `test-ssr`. The dependency is found correctly in root node_modules, but then its dependencies like `@fluentui/react-jsx-runtime/jsx-runtime` point towards a lib folder that is not being generated in node_modules.
image

any clues @Hotell or @layershifter?

**Review by JustSlone — APPROVED** [review]
Signing off now on the codeowner change

**Comment by layershifter** [comment]
> Ideal solution is to have probably both so until that type-checking epic is done I created esbuild plugin to add ts path resolution back -> everything works as before

@Hotell agree with your proposal 👍 Thx for looking into that ❤️

**Comment by Hotell** [comment]
@sopranopillow Once these 2 PRs land to master , update your branch to get unblocked 

- https://github.com/microsoft/fluentui/pull/31226
- https://github.com/microsoft/fluentui/pull/31227

**Comment by Hotell** [comment]
> @sopranopillow Once these 2 PRs land to master , update your branch to get unblocked
> 
> * [feat(scripts-storybook): make getMetadata throwable via config to be able to process stories glob for packages no longer present in the repo #31226](https://github.com/microsoft/fluentui/pull/31226)
> * [feat(test-ssr): add custom esbuild-plugin to properly resolve TS path aliases which stopped working starting esbuild 0.19 #31227](https://github.com/microsoft/fluentui/pull/31227)

both landed to master, now your PR is unblocked 🥳 !

**Review by TristanWatanabe — APPROVED** [review]

**Pull request merged**
